components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -20,6 +22,22 @@
 CLP_METADATA_TABLE_PREFIX = 'clp_'
 SEARCH_JOBS_TABLE_NAME = 'distributed_search_jobs'
 
+# Package modes
+class PackageMode(KebabCaseStrEnum):
```

Comment:
As we were discussing offline, other option names are:

* PackageFlavor [sic]
* CompressionEngine
* CoreEngine

PackageMode kind of implies a small config change whereas switching between clp and clp-s seems more significant.

Reply:
Ended up going with StorageEngine which implies a big configuration change, and is a bit more generic than CompressionEngine